samples/04_gis_analysts_data_scientists/identifying-county-names-from-incomplete-house-addresses.ipynb

```diff
@@ -0,0 +1,1903 @@
+{
```

Comment:
<p>Description talks about an error rate, but that is not seen in the output.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/798/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Reply:
fixed the description